test/unit/python-requirements-parser.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { getRequirements } from "../../lib/python-parser/requirements-parser";
+
+describe("python requirements parser", () => {
+  it("correctly parses a requirements file", () => {
+    const fileContent = `
+      Jinja2==2.7.2
+      oauth2>=1.1.3
+      pyramid==1.3a6
+    `;
+    const requirements = getRequirements(fileContent);
+    expect(requirements).toHaveLength(3);
+    expect(requirements).toMatchObject([
+      { name: "jinja2", specifier: "=", version: "2.7.2" },
+      { name: "oauth2", specifier: ">=", version: "1.1.3" },
+      { name: "pyramid", specifier: "=", version: "1.3a6" },
+    ]);
+  });
+
+  it("parses package names without versions", () => {
+    const fileContent = `
+      jinja2==2.7.2
+      oauth2
+      pyramid~=1.3a6
+    `;
+    const requirements = getRequirements(fileContent);
+    expect(requirements).toHaveLength(3);
+    expect(requirements).toMatchObject([
+      { name: "jinja2", specifier: "=", version: "2.7.2" },
+      { name: "oauth2" },
```

Comment:
What happens in this case where we don't have a version?

Reply:
https://github.com/snyk/snyk-docker-plugin/pull/443/files#diff-c2e9b021b0d027e8d361854e722e693c22bcc7f95a72cc146739d0a05d117826R64
We will take the latest version we find for this package. Or at least that was the intention, but looking at it I found a bug (the sorting comes after). I'll fix and add a test case.